semantic/src/site/globals/site.overrides

```diff
@@ -325,12 +321,16 @@ a:hover {
     padding-top: 8px;
     padding-bottom: 8px;
   }
+  @media only screen and (max-width: 860px) {
```

Comment:
The 860 is just an arbitrary amount based on what seemed reasonable on my screen.